### PR TITLE
Reedit of PGD doc - group 13

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/upgrades/tpa_overview.mdx [hunk 1]
```diff
@@ -24,11 +24,11 @@ Major version upgrades of PGD require changes to the TPA `config.yml` file, whic
 
 When upgrading to PGD 5 from previous PGD major versions, you can use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/). This command helps you make appropriate modifications to your deployment configuration. 
 
-The `reconfigure` command requires settings for architecture (only `PGD_Always_ON` and PGD Proxy routing to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration. 
+The `reconfigure` command requires settings for architecture-only `PGD_Always_ON` and PGD Proxy routing to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/upgrades/tpa_overview.mdx:27]: @djw-m Please check this edit. There was an errant parentheses and I'm not sure that the edit is correct.

Comment (djw-m) [reply product_docs/docs/pgd/5/upgrades/tpa_overview.mdx:27]:
```suggestion
The `reconfigure` command requires settings for architecture (the only supported setting is `PGD_Always_ON`) and PGD Proxy routing (`--pgd-proxy-routing <global|local>`) to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration. 
```

File: product_docs/docs/pgd/5/security/roles.mdx [hunk 2]
```diff
@@ -16,16 +16,16 @@ PGD are split across the following predefined roles.
...
 The PGD `bdr_` roles are created when the BDR extension is installed. See [PGD
 predefined roles](pgd-predefined-roles) for more details of the privileges each
-one has.
+role has.
 
 Managing PGD doesn't require that administrators have access to user data.
 
 Arrangements for securing information about conflicts are discussed in
-[Logging conflicts to a table](../reference/conflict_functions/#logging-conflicts-to-a-table).
+[Logging conflicts to a table](pgd/latest/reference/conflict_functions/#logging-conflicts-to-a-table).
```
Comment (djw-m) [inline product_docs/docs/pgd/5/security/roles.mdx:28]:
```suggestion
[Logging conflicts to a table](pgd/latest/reference/conflict_functions#logging-conflicts-to-a-table).
```

File: product_docs/docs/pgd/5/security/roles.mdx [hunk 3]
```diff
@@ -16,16 +16,16 @@ PGD are split across the following predefined roles.
...
 The PGD `bdr_` roles are created when the BDR extension is installed. See [PGD
 predefined roles](pgd-predefined-roles) for more details of the privileges each
-one has.
+role has.
 
 Managing PGD doesn't require that administrators have access to user data.
 
 Arrangements for securing information about conflicts are discussed in
-[Logging conflicts to a table](../reference/conflict_functions/#logging-conflicts-to-a-table).
+[Logging conflicts to a table](pgd/latest/reference/conflict_functions#logging-conflicts-to-a-table).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/security/roles.mdx:28]:
```suggestion
[Logging conflicts to a table](pgd/latest/reference/conflict_functions/#logging-conflicts-to-a-table).
```

File: product_docs/docs/pgd/5/security/roles.mdx (hunk 2 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/security/roles.mdx:28]:
```suggestion
[Logging conflicts to a table](pgd/latest/reference/conflict_functions#logging-conflicts-to-a-table).
```

File: product_docs/docs/pgd/5/security/roles.mdx (hunk 3 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/security/roles.mdx:28]:
```suggestion
[Logging conflicts to a table](/pgd/latest/reference/conflict_functions#logging-conflicts-to-a-table).
```

File: product_docs/docs/pgd/5/routing/readonly.mdx
Comment (djw-m) [inline product_docs/docs/pgd/5/routing/readonly.mdx:1]: The read-only file had edits in Group 12 previously. Not sure why its in this group too.

**Review by djw-m — APPROVED** [review]
Apart from the doubling up on one file, LGTM.

**Pull request merged**